plugins/inputs/glusterfs/glusterfs.go

```diff
@@ -0,0 +1,71 @@
+package glusterfs
+
+// glusterfs.go
+
+import (
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	"bufio"
+	"os/exec"
+	"regexp"
+	"strconv"
+)
+
+var matchBrick = regexp.MustCompile("^Brick: (.*)$")
+var matchRead = regexp.MustCompile("Data Read: ([0-9]+) bytes$")
+var matchWrite = regexp.MustCompile("Data Written: ([0-9]+) bytes$")
+
+type GlusterFS struct {
+	Volumes []string
+}
+
+func (gfs *GlusterFS) Description() string {
+	return "Plugin reading values from the GlusterFS profiler"
+}
+
+func (gfs *GlusterFS) SampleConfig() string {
+	return "volumes = [\"volume-name\"]"
+}
+
+func (gfs *GlusterFS) Gather(acc telegraf.Accumulator) error {
+	for _, volume := range gfs.Volumes {
+		var cmdName = "sudo"
```

Comment:
Yes, I don't like it either, just don't know how to avoid it. telegraf dones't run as root, the gluster command doesn't work if you are not root, and I'm not aware of anything I can do about that. I'll post on the ML, to check